packages/subgraph/src/util.ts

```diff
@@ -1,9 +1,21 @@
-import { BigInt } from "@graphprotocol/graph-ts";
+import { Address, BigInt, dataSource } from "@graphprotocol/graph-ts";
 import { Account } from "../generated/schema";
 
-export const BIGINT_ONE = BigInt.fromI32(1)
-export const BIGINT_ZERO = BigInt.fromI32(0)
+export const BIGINT_ONE = BigInt.fromI32(1);
+export const BIGINT_ZERO = BigInt.fromI32(0);
+export const SHORT_HELPER_ADDR = getShortHelperAddr(dataSource.network());
 
+function getShortHelperAddr(networkName: string): Address {
+  let addr = Address.fromString("0x3b4095D5ff0e629972CAAa50bd3004B09a1632C5");
```

Comment:
do we wanna make these addresses constants?

Reply:
Okay, will do